<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_s1</class>
 <widget class="QDialog" name="update_s1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="textEdit_userNo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UserNo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_userNo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Check UserNo</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_balance">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>200</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_amount">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_recharge">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>200</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Recharge</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
